src/test/java/nextstep/subway/documentation/PathDocumentation.java

```diff
@@ -1,19 +1,30 @@
 package nextstep.subway.documentation;
 
-import io.restassured.RestAssured;
+import static nextstep.subway.acceptance.PathSteps.경로_조회_예시_응답;
+import static nextstep.subway.acceptance.PathSteps.두_역의_최단_거리_경로_조회를_요청;
+import static nextstep.subway.acceptance.PathSteps.출력_필드_추가;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.when;
+
+import nextstep.subway.applicaion.PathService;
+import nextstep.subway.applicaion.dto.PathResponse;
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.springframework.http.MediaType;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+class PathDocumentation extends Documentation {
 
-public class PathDocumentation extends Documentation {
+    @MockBean
+    PathService pathService;
 
+    @DisplayName("경로 조회 문서")
     @Test
-    void path() {
-        RestAssured
-                .given().log().all()
-                .accept(MediaType.APPLICATION_JSON_VALUE)
-                .queryParam("source", 1L)
-                .queryParam("target", 2L)
-                .when().get("/paths")
-                .then().log().all().extract();
+    void findPath() {
+        PathResponse pathResponse = 경로_조회_예시_응답();
```

Comment:
```suggestion
        var pathResponse = 경로_조회_예시_응답();
```

타입 추론을 사용해보는 것은 어떤가요?